src/Tribe/Views/V2/View.php

```diff
@@ -1568,4 +1569,44 @@ protected function filter_display_events_bar( $display ) {
 
 		return $display;
 	}
+
+	/**
+	 * Returns the iCal data we're sending to the view.
+	 *
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	protected function get_ical_data() {
+		/**
+		 * A filter to control whether the "iCal Import" link shows up or not.
+		 *
+		 * @since unknown
+		 *
+		 * @param boolean $show Whether to show the "iCal Import" link; defaults to true.
+		 */
+		$display_ical = apply_filters( 'tribe_events_list_show_ical_link', true );
+
+		/**
+		 * Allow for customization of the iCal export link "Export Events" text.
+		 *
+		 * @since unknown
+		 *
+		 * @param string $text The default link text, which is "Export Events".
+		 */
+		$link_text  = apply_filters( 'tribe_events_ical_export_text', __( 'Export Events', 'the-events-calendar' ) );
+
+		$link_title = __( 'Use this to share calendar data with Google Calendar, Apple iCal and other compatible apps', 'the-events-calendar' );
+
+		$ical_data = [
+			'display_link' => $display_ical,
+			'link'         => (object) [
+				'url'    => esc_url( tribe_get_ical_link() ),
+				'anchor' => $link_text,
```

Comment:
can we change this to `'label'` or something that's a bit clearer? was there a specific reason for naming it `'anchor'`? to me, `'anchor'` means the link or url and had me confused.

Reply:
I would go with `link_url`, `link_text` and `link_title`